src/workflow/workers/indexing.py

```
@@ -0,0 +1,200 @@
+
+        Args:
+            episode: Episode to index.
+
+        Returns:
+            Tuple of (resource_name, display_name).
+
+        Raises:
+            Exception: If indexing fails.
+        """
+        if not episode.transcript_path:
+            raise ValueError(f"Episode {episode.id} has no transcript_path")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)